config/ufs/machines/config_machines.xml

```diff
@@ -428,6 +428,65 @@ This allows using a different mpirun command to launch unit tests
     </environment_variables>
   </machine>
 
+  <machine MACH="jet">
+    <DESC>NOAA JET system</DESC>
+    <NODENAME_REGEX>fe\d?.fsl.noaa.gov</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <COMPILERS>intel</COMPILERS>
+    <MPILIBS>impi</MPILIBS>
+    <PROJECT>wrfruc</PROJECT>
+    <CIME_OUTPUT_ROOT>$ENV{CIME_OUTPUT_ROOT}</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>$ENV{DIN_LOC_ROOT}/ufs_inputdata</DIN_LOC_ROOT>
```

Comment:
I suggested that you make CIME_OUTPUT_ROOT an environment variable. 
Making all of these environment variables means there is a potential to duplicate a lot of stuff on jet that you don't need to duplicate. It also means that each jet user will have to have a set of instructions for setting up the system - is there no way to get a shared filesystem there?